app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: note, local: true) do |form| %>
+  <% if note.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(note.errors.count, "error") %> prohibited this note from being saved:</h2>
```

Comment:
`<%= pluralize(note.errors.count, "error") %> prohibited this note from being saved:` could also become a parameterized I18n string